### DISCUSSION TITLE
How do I change the OneDrive Root Mount location?

### DISCUSSION THREAD

**Discussion by cventophd** [abraunegg/onedrive#2450] [Q&A] [answered]
Hello,

I need to change the location of the OneDrive rount mount. It is currently in my systems root file system, but I need to move it one of the external hard-drives. What specifically do I need to do that? Ideally - I would to simply to change the root directory without needing to over-ride anything, since I have already moved the files in the original root drive directory to the external hard drive. 

Thanks.

**Comment by abraunegg** [comment, marked as answer]
@cventophd 
> How do I change the OneDrive Root Mount location?

This client does not 'mount' your OneDrive account. This client syncs your data and local changes between OneDrive and your local system. 

Where this data is stored on your local system is governed by the 'sync_dir' config file option. Please read the documentation and discussion posts in relation to this config option.

> but I need to move it one of the external hard-drives

External hard drive as in connected via a USB or other like eSATA .. or an internal SATA cable where the device cannot be accidentally 'bumped' leading to unexpected removal?

If truly external, then you also are *highly* advised that you read the following advice: https://github.com/abraunegg/onedrive/blob/master/docs/advanced-usage.md#configuring-the-client-for-use-when-sync_dir-is-a-mounted-directory
